modules/openapi-generator/src/main/resources/C-libcurl/apiClient.c.mustache

```diff
@@ -530,7 +530,7 @@ void apiClient_invoke(apiClient_t    *apiClient,
 
         res = curl_easy_perform(handle);
 
-        curl_slist_free_all(headers);
+        curl_slist_freeList_all(headers);
```

Comment:
I don't think `curl_slist_free_all` should be modified because it is a function of libcurl https://curl.se/libcurl/c/curl_slist_free_all.html

@minerba 
please create a new PR to rollback these changes for `curl_slist_free_all`

Reply:
@ityuhui 
Ok I will create a new PR soon.